conf/messages.en

```diff
@@ -140,3 +137,21 @@ researchBanner.heading = Help improve GOV.UK
 researchBanner.inviteLink = Help improve digital services by joining the HMRC user panel (opens in a new window)
 researchBanner.closeLink = No thanks
 
+# Landing Page
+# ----------------------------------------------------------
+view.start.title=Trader services
```

Comment:
Proto has this has "Use trader services" but maybe Trader services is more appropriate?